tee.go

```diff
@@ -3,10 +3,8 @@ package main
 import (
 	"bufio"
 	"bytes"
-	"fmt"
-	"io"
-
 	"github.com/mattn/go-colorable"
+	"io"
```

Comment:
I mean this!

this should be like this:

```go
	"io"	

	"github.com/mattn/go-colorable"
```

Reply:
done :)